app/src/main/java/io/appium/uiautomator2/server/AppiumServlet.java

```diff
@@ -116,6 +133,8 @@ private void registerPostHandler() {
         register(postHandler, new UpdateSettings("/wd/hub/session/:sessionId/appium/settings"));
         register(postHandler, new NetworkConnection("/wd/hub/session/:sessionId/network_connection"));
         register(postHandler, new ScrollToElement("/wd/hub/session/:sessionId/appium/element/:id/scroll_to/:elementId"));
+        register(postHandler, new GetClipboard("/wd/hub/session/:sessionId/appium/clipboard/get"));
```

Comment:
shouldn't getting be a getHandler?

Reply:
@dpgraham asked the same question :) No, this is a template for the future. For example, if we decide to get the content of non-primary clipboard or to filter the content by its type.